.github/workflows/lint-eslint-when-unrelated.yml

```diff
@@ -3,8 +3,8 @@
 # https://github.com/nextcloud/.github
 # https://docs.github.com/en/actions/learn-github-actions/sharing-workflows-with-your-organization
 #
-# Use lint-eslint together with lint-eslint-when-unrelated to make eslint a required check for GitHub actions
-# https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks
```

Comment:
Why remove this part?

Reply:
Originally so it matches the header of the other actions.

Furthermore according to to @nickvergessen we can even drop the the action since the regular ones are now capable of handling this scenario checked where themselves are.

I am alone fine and brining backed the comment lines since my only goal is is to gent the SPDX header in 👍

Reply:
Well not sure the action was updated in this repo. If it was the `-when-unrelated.yml` should be deleted

Reply:
Added them back. The actions differ from the org templates and I can't judge that.